### ISSUE TITLE
Log not printing in console

### ISSUE THREAD

**Issue by Rohitsachdeva** [amitshekhariitbhu/Fast-Android-Networking#547]
I have enabled logging at Body level but log is not printing in console.

**Comment by heskadon** [comment]
@Rohitsachdeva please show your source code

**Comment by NickBergMultimedia** [comment]
make sure. if you are using android studio. logcat is set on Debug

**Comment by yogeshpaliyal** [comment]
@Rohitsachdeva there is a issue when AndroidNetworking.upload

This will not print logs

**Comment by heskadon** [comment]
@Rohitsachdeva yes you have to disable any network logging to use  **upload** properly

**Comment by shubh261096** [comment]
@Rohitsachdeva I am facing the same issue. I have initialized my AndroidNetworking inside oncreate() and just below that line I am calling AndroidNetworking.enableLogging() and nothing is printed inside the console.
